app/models/multi_checksum.rb

```diff
@@ -3,4 +3,13 @@ class MultiChecksum < Valkyrie::Resource
   attribute :sha256, Valkyrie::Types::SingleValuedString
   attribute :md5, Valkyrie::Types::SingleValuedString
   attribute :sha1, Valkyrie::Types::SingleValuedString
+
+  def self.factory(file_object)
```

Comment:
We've used `.for` for the Factory Method pattern before. I don't really care which we use, but it should match https://github.com/pulibrary/figgy/blob/master/app/models/file_metadata.rb#L15

Reply:
i like `.for`; sounds good.